<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receiver</class>
 <widget class="QWidget" name="Receiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receiver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>host:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hostLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portLlineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connectPushButton">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>recvLength:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lenLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>base:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="basePlainTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>received bits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="bitsPlainTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>received bases:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="recvBasePlainTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Final Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="keyPlainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
